vscodeLogo/generate.cmd

```diff
@@ -0,0 +1,29 @@
+setlocal enabledelayedexpansion
+@echo off
+
+rem generate png from svg using inkscape
+
+rem svg
+set name=logo
+
+set THISDIR=%~dp0
+set THISDIR=%THISDIR:~,-1%
+
+rem inkscape program path
+set inkscape="%tools%\Programs\inkscape\inkscape.exe"
```

Comment:
To get this to work:

1. Install [inkscape](https://inkscape.org/)
2. Replace `"%tools%\Programs\inkscape\inkscape.exe"` with the location inkscape on your machine.